addons/survey/models/survey_survey.py

```diff
@@ -583,9 +583,32 @@ def _get_pages_or_questions(self, user_input):
             else:
                 result = self.question_and_page_ids.filtered(
                     lambda question: not question.is_page or not is_html_empty(question.description))
+                result = self._get_pages_and_questions_to_show(result)
 
         return result
 
+    @api.model
+    def _get_pages_and_questions_to_show(self, questions):
+        """
+        This method takes a survey.question recordset and
+        excludes conditional questions that have an invalid trigger.
+
+        :param questions: survey.question recordset (includes pages that have a description and questions)
+        :return: survey.question recordset excluding invalid conditional questions
+        """
```

Comment:
Thank you for your continued efforts and good work, Adrien!

I notice here, as last comment, that you don't mention the empty description pages here and that the `:param:` and `:return:` info is the same as the docstring description (looks like there is a mixup with the parentheses too). 

In any case, the docstring could be made more concise by avoiding repetition, using either the description or the `:` part could be enough.

Finally, if you are confident that this final version is improved with respect to the original draft,feel free to squash your commits so the reviewer sees your final PR and commit message :)